BridgeCareApp/VuejsApp/src/components/investment-editor/InvestmentEditor.vue

```diff
@@ -0,0 +1,604 @@
+<template>
+    <v-container fluid grid-list-xl>
+        <div style="height: 850px; overflow-x: hidden; overflow-y: auto;">
```

Comment:
We can create a class for the styles and use it here. It can be in the same .vue file after the script tag.